src/CMakeLists.txt

```diff
@@ -65,13 +69,6 @@ ExternalProject_Add(libderiv_library
                -DLIBC_INTERJECT=${LIBC_INTERJECT}
                -DLibintint_DIR=${STAGED_INSTALL_PREFIX}/share/cmake/Libint
                -DMERGE_LIBDERIV_INCLUDEDIR=${MERGE_LIBDERIV_INCLUDEDIR}
-    CMAKE_CACHE_ARGS -DCMAKE_C_FLAGS:STRING=${CMAKE_C_FLAGS}
-                     -DCMAKE_CXX_FLAGS:STRING=${CMAKE_CXX_FLAGS}
-    DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/libderiv-src
-    DOWNLOAD_COMMAND ${STAGED_INSTALL_PREFIX}/bin/libderiv_compiler
-    LOG_DOWNLOAD 1
-    BINARY_DIR ${CMAKE_BINARY_DIR}/libderiv-src
-    INSTALL_COMMAND DESTDIR=${CMAKE_BINARY_DIR}/stage ${CMAKE_MAKE_PROGRAM} install
```

Comment:
All the install commands removed? I think you made it less of a superbuild build-dir scheme. I'll run it to see if I notice any dire implications.

Reply:
There is no `make` command on Windows, so we will have to replace this here and *Psi4* repo.

Reply:
Could you use Ninja instead of `make`? @robertodr ported a lot of the ecosystem builds to Ninja, and [this](https://blogs.msdn.microsoft.com/vcblog/2017/06/14/cmake-support-in-visual-studio-whats-new-in-2017-15-3-preview-2/) looks like it's Windows-friendly. Changing up the build-dir structure is probably do-able for Libint, but it sounds alarming for Psi4 itself (though by your outputs it does seem to work).

Reply:
Currently I'm using *CMake* and Windows-native *MSBuild* system. *Ninja* should also work, if *CMake* files are build system agnostic.

For *Psi4*, it is done, just need some clean up and will make a PR.

Reply:
By removing the `INSTALL_COMMAND`, the build dir has structure `stage/bin-lib-include/` instead of `stage/CMAKE_INSTALL_PREFIX/bin-lib-include`. Since (1) one never runs libint from the build dir (no tests) and (2) the final install isn't a true superbuild install of `cp -R staged/ /`, this works just fine. I still have reservations about how well this would work for Psi4; I suspect little things around the fringes would break.

Psi4's CMake should work with GNUMake and Ninja generators. I'm still confused why `CMAKE_MAKE_PROGRAM` isn't defined for you. According to [this](https://cmake.org/cmake/help/v3.12/variable/CMAKE_MAKE_PROGRAM.html), it should be set to `MSBuild.exe`. 
 Does `INSTALL_COMMAND DESTDIR=${CMAKE_BINARY_DIR}/stage ${CMAKE_VS_MSBUILD_COMMAND} install` work any better?

Other than trying to avert Psi4 build troubles by fixing the CMAKE_MAKE_PROGRAM trouble here, lgtm!

Reply:
Thanks for the fuller explanation, @raimis. @robertodr, have you come across any pattern for Windows that can get around these points? I'm worried about the `cp -R stage/ /` step for psi4. If there are relative links that go out of the project and into the filesystem, they'll get confused if CMAKE_PREFIX_PATH isn't under `objdir/stage`, I think.

Reply:
I'm going to go ahead and start the Valeev process, but if you get a chance to look at this, @robertodr, I'd appreciate it.